newspaper/outputformatters.py

```diff
@@ -141,9 +141,12 @@ def remove_empty_tags(self):
     def remove_trailing_media_div(self):
         """Punish the *last top level* node in the top_node if it's
         DOM depth is too deep. Many media non-content links are
-        eliminated: "related", "loading gallery", etc
+        eliminated: "related", "loading gallery", etc. It skips removal if
+        last top level node's class is one of NON_MEDIA_CLASSES.
         """
 
+        NON_MEDIA_CLASSES = ('zn-body__read-all', )
```

Comment:
I merged your other commit, can you rebase so these changes don't show?

Reply:
sure thing - changed!